app/models/host/base.rb

```diff
@@ -250,6 +250,10 @@ def bond_interfaces
       self.interfaces.bonds.is_managed.all
     end
 
+    def bridges_interfaces
```

Comment:
Typo, `bridge_interfaces`. Add it to the `Jail` on [Host::Managed L62](https://github.com/theforeman/foreman/blob/develop/app/models/host/managed.rb#L62), it's the whole purpose of all these `xxxx_interfaces` methods I think.